app/components/job/details/error-display/job-error-display.html

```diff
@@ -0,0 +1,16 @@
+<bex-banner *ngIf="jobFailed" fixMessage="View failed tasks" [fix]="gotoFailedTask" type="warning">
+    <div message>Job was terminated because one task failed.</div>
```

Comment:
well as soon as a task failed it gets terminated so there should only be 1 task that cause the job to fail. But I agree should be changed